doc/source/getting-started-linux-tutorial.rst

```diff
@@ -79,85 +81,78 @@ Thanks go to `stuxnet <https://github.com/stuxnet999/>`_ for providing this memo
+Once created, place the file under the ``volatility3/symbols`` directory so that Volatility3 can recognize it automatically.
+
+linux.boottime
+~~~~~~~~~~~~~~
+
+This plugin provides the system boot time extracted from memory.  
+It is useful for establishing a timeline, particularly when analyzing incident response scenarios or determining system uptime.
+
+.. code-block:: shell-session
+
+    $ python3 vol.py -f memory.vmem linux.boottime
+
+        Volatility 3 Framework 2.26.0
+        Progress:  100.00               Stacking attempts finished
+
+        TIME NS Boot Time
+
+        -       2022-02-10 06:50:16.450008 UTC
+
+This timestamp can serve as a reference point for correlating system events, such as process start times, logs, or malicious activity.
 
-.. tip:: Use the banner text which is most repeated to search on the ISF Server.
 
 linux.pslist
 ~~~~~~~~~~~~
 
+This plugin lists active processes by walking the task list from memory.  
+It provides detailed metadata for each process, including identifiers and user/group information.
+
 .. code-block:: shell-session
 
     $ python3 vol.py -f memory.vmem linux.pslist
 
-        Volatility 3 Framework 2.0.1    Stacking attempts finished
-
-        PID     PPID    COMM
-
-        1       0       systemd
-        2       0       kthreadd
-        3       2       kworker/0:0
-        4       2       kworker/0:0H
-        5       2       kworker/u256:0
-        6       2       mm_percpu_wq
-        7       2       ksoftirqd/0
-        8       2       rcu_sched
-        9       2       rcu_bh
-        10      2       migration/0
-        11      2       watchdog/0
-        12      2       cpuhp/0
-        13      2       kdevtmpfs
-        14      2       netns
-        15      2       rcu_tasks_kthre
-        16      2       kauditd
-        .....
-
-``linux.pslist`` helps us to list the processes which are running, their PIDs and PPIDs.
+        Volatility 3 Framework 2.26.0
+        Progress:  100.00               Stacking attempts finished
+        OFFSET (V)      PID     TID     PPID    COMM    UID     GID     EUID    EGID    CREATION TIME   File output
+
+        0x8ca6db1aac80  1       1       0       systemd 0       0       0       0       2022-02-10 06:50:16.364213 UTC  Disabled
+        0x8ca6db1a9640  2       2       0       kthreadd        0       0       0       0       2022-02-10 06:50:16.364213 UTC  Disabled
+        0x8ca6db1ac2c0  3       3       2       rcu_gp  0       0       0       0       2022-02-10 06:50:16.372213 UTC  Disabled
+        ...
+
+This detailed view allows investigators to correlate user privileges, startup times, and relationships between processes more precisely than before.
+
 
 linux.pstree
 ~~~~~~~~~~~~
 
+This plugin presents the process hierarchy as a tree, clearly showing parent-child relationships between processes.  
```

Comment:
Hi @ikelos, @eve-mem 
This section explains the usage and meaning of the plugin.
But I think it should focus on the simple feature in that document.
I think the explanation that analyzes this in detail can be explained in more detail later in the malware plugin.
(`volatility/framework/plugin/linux/malware`)
What do you think about this?